karapace/schema_backup.py

```diff
@@ -24,10 +25,27 @@
 LOG = logging.getLogger(__name__)
 
 
+JsonData = Any
```

Comment:
Please use the typing:

https://github.com/aiven/karapace/blob/eb5a7720475b66de5011722db4799e3815fdacaa/karapace/typing.py#L3

Reply:
Done.